setup.cfg

```diff
@@ -0,0 +1,4 @@
+[isort]
+multi_line_output=3
```

Comment:
line-width is missing. And, if you like, you can order the options alphabetically.